kommunicate/src/main/java/com/applozic/mobicomkit/api/notification/NotificationService.java

```diff
@@ -51,6 +55,7 @@
  * Time: 7:36 PM
  */
 public class NotificationService {
+    public static final String NOTIFICATION_TEXT_NOT_AVAILABLE = "You received a message";
```

Comment:
Are we using the same in Web?

Reply:
No but rajeesh sir told to use this string as default